iot-lambda-pub-receiver-cdk/README.md

```diff
@@ -0,0 +1,58 @@
+# AWS Lambda to AWS IoT Core
+
+This pattern deploys one publisher AWS Lambda, which publishes a message in AWS IoT Core topic. The topic is watch by a rule which triggers an action when the condition is met. The action calls a AWS Lambda.
+
+Learn more about this pattern at Serverless Land Patterns: << Add the live URL here >>
```

Comment:
Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/iot-lambda-pub-receiver-cdk